build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
Is it possible for us to submit a "precompile coreroot job", after core_root is uploaded? This will ensure that subsequent jobs, which will run tests, do not have to precompile the core_root.

Also, have you tested the R2R test run locally and confirmed that the tests are getting compiled on Windows (which is what your PR is for)?